pkg/registry/core/bastion/bastion_suite_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Why was the `mock/doc.go` file removed? 👀

Reply:
That's an excellent question, I certainly did not do this on purpose. Let me check.

Reply:
I can only explain it a momentary "surely everything in pkg/client is generated, let's just rm -rf it and regenerate it" lapse in conscience. I've restored the files now.

Reply:
Hm, I still don't see them 👀